### ISSUE TITLE
URGENT AND SUPER IMPORTANT: MC Charges

### ISSUE THREAD

**Issue by ali206** [206Tours/DB#1111]
I know we've been emailing about this but this really needs to be fixed ASAP.  

Maybe we need to have a call about this with Milanka?

![image](https://cloud.githubusercontent.com/assets/8440903/14862013/ac11d78a-0c7d-11e6-8f1c-bf766b70958e.png)

**Comment by bradatkins** [comment]
Hey -

I'm OK with a call...I'll change the DB to grab the exchange rate at time of payment.

I'm not sure on the 21.21 as I never see that?

**Comment by ali206** [comment]
Ok don't change anything until Milanka confirms please

**Comment by ali206** [comment]
Ok i checked with Milanka, please change the DB to grab the exch rate at time of payment

**Comment by bradatkins** [comment]
Ok it's done but I will be further testing it

**Comment by bradatkins** [comment]
OK I just did a test on 423410. I did a payment of $100 CAD, and it grabbed the exchange rate of the current time. It converted it to $74 USD (ouch)

**Comment by ali206** [comment]
Ok that's good news! We'll continue to monitor.

the base price isn't matching the website though in CAD and the pre-tour remained $1,200 no matter what currency i was looking at it in ( I know it's a test booking but still?)

![image](https://cloud.githubusercontent.com/assets/8440903/14965185/f28c87aa-1079-11e6-8f79-55f84600b23d.png)


![image](https://cloud.githubusercontent.com/assets/8440903/14965181/ebefd410-1079-11e6-9a6f-16f55f7e06df.png)

**Comment by ali206** [comment]
Ok so this is super confusing - let me know if i'm not clear enough...

As i mentioned earlier, what is posting to our bank account is not the $ amount our clients are seeing on their statement and is not totally matching the DB either all the time due to Interchange dollars (fee to us). 

for example: 423227
She made a payment on 4/6/16 for $4,324.32 CAD which Elavon converted in to USD at an exch rate of .74519 = $3,222.43 USD. The Interchange Dollars are $58.00 USD so the total the client should be seeing on their cc is **$3,280.43 USD**.  

![image](https://cloud.githubusercontent.com/assets/8440903/14966131/dbfd1540-107e-11e6-9c3a-31e67bdff985.png)

![image](https://cloud.githubusercontent.com/assets/8440903/14966147/ea51a57a-107e-11e6-976b-7f1b5b966883.png)



However, the DB is converting USD in to CAD at an exch rate of 1.32 which on her invoice is showing as **$3,276 USD**
![image](https://cloud.githubusercontent.com/assets/8440903/14966051/8f171cee-107e-11e6-88c3-81325f391222.png)

![image](https://cloud.githubusercontent.com/assets/8440903/14966068/9b1df04e-107e-11e6-98af-041444643ff3.png)

So now, the DB is not matching the bank which is not matching Elavon and i don't know what to do about that. 

I think the DB should be converting CAD in to USD like Elavon is and then hopefully it will be around the same exch rate and closer to payment client is seeing.

How do other companies do this??

**Comment by ali206** [comment]
Also, Elavon is converting CAD in to USD at different times then when the transaction is processed so the exch rate will never be exact....

**Comment by ali206** [comment]
Maybe you can do a test booking with one of your CC's and then let me know what you're seeing on your statement? we can refund you of course

I don't want to pay the fees on that charge but i don't know how else to confirm/show this

**Comment by bradatkins** [comment]
Elavon does not return anything to me, from what I know, about the Interchange change. Their documentation never mentions it. Can you call them and see? I find it odd that they would add this (and if they do add it that they would charge the PAX this value and not 206 Tours as the PAX never agrees to or sees this)

On the exchange rate, I agree. Any other CC company I've ever worked with does all the FX. We discussed this at length, which still does not align to what the rest of the industry does.

What should happen is I send Elavon $X CAD and they charge $X CAD and tell me that's $Y USD. That's how authorize.net (one of the leading companies does it).

I tried calling you today but just missed you. We can chat over the phone if you'd like.

**Comment by ali206** [comment]
What should happen is I send Elavon $X CAD and they charge $X CAD and tell me that's $Y USD. That's how authorize.net (one of the leading companies does it).

ok and why is elavon not communicating with DB on that again?

I'm here til 5 if you want to call again

**Comment by bradatkins** [comment]
I'm not sure...that's just how Elavon works (or doesn't)

Even on the Installments thing, it's disjointed. Their whole setup is truly
odd.

**Comment by ali206** [comment]
Ok, very annoying!  We could do this through paypal but they only accept 6 different currencies (not sure which ones) but there are sooo many fees through them.  That's the best thing about Elavon was that there were no associated fees for the MCC.  

Do you know of other companies we could use for multi currency that might work better with the DB?

**Comment by ali206** [comment]
I asked someone at Elavon to explain the interchange more and this is what he said:

Ali,
“Interchange” is the industry term for the wholesale charge from the credit card companies….V/MC/Discv….this is not a fee from Elavon.

Also you want me to call Elavon and ask what?  

**Elavon does not return anything to me, from what I know, about the Interchange change. Their documentation never mentions it. Can you call them and see?**

The odd thing to me is that only 1 client has called to complain about $ amount discrepancy.  so how do i move forward? Should i just edit the $ amount the DB is converting to be what shows in the Elavon statement?  Seems silly but i don't know what else to do right now

**Comment by bradatkins** [comment]
There are fees for MCC With ELAVON...those interchange ones?

**Comment by ali206** [comment]
Those aren't being charged by Elavon though, they're coming from the CC companies like VI/MC

**Comment by bradatkins** [comment]
If they aren't charged by Elavon (the interchange fee) how is the DB supposed to find out about them?

**Comment by ali206** [comment]
Exactly...that's my question as well.  I don't know how to make this MCC function work correctly with the DB

**Comment by ali206** [comment]
I just spent the last hour on the phone with Elavon....

I just don't know a solution to this issue...

The guy from Elavon told me they're converting at the same time we are, but they're pulling the exch rate from the database they receive at the end of the day at 6:00 pm East Std time....so i dont know why he kept saying that they're converting the same time as we are.

They're grabbing their exch rate from the int'l exch database

So even if they could communicate with our DB it's to late anyway b/c the client needs to see what they'e being charged at that exact moment. 

So from now on I will have to check amount in DB to Elavon statement and adjust DB according to what (i believe) should be showing on the clients card?? They're not to far off but they're not matching. Right? Or what else do I do...
